### DISCUSSION TITLE
returning extreme large value

### DISCUSSION THREAD

**Discussion by 1Euro7Cent** [infinitered/nsfwjs#539] [Q&A]
image: https://cdn.discordapp.com/attachments/721821364255719504/880540077850247198/unknown.png

returnings:
```js
[
  { className: 'Neutral', probability: 0.9999781847000122 },
  { className: 'Drawing', probability: 0.00002110443529090844 },
  { className: 'Porn', probability: 4.6671684117427503e-7 },
  { className: 'Hentai', probability: 2.755132015863637e-7 },
  { className: 'Sexy', probability: 2.7379114087011658e-8 }
]
```

**Comment by GantMan** [comment]
Ahhh, the issue is that it's represented using scientific notation.   See the `e-7`? and `e-8`? that actuall makes the number VERY small, like  0.000000432

https://calculator.name/scientific-notation-to-decimal/4.32e-7

This is a common notation and shouldn't be an issue for finding max values.

**Comment by 1Euro7Cent** [comment]
oh. i didnt know that. but still thank you

**Comment by GantMan** [comment]
If you have any issues with it being in this form, please update me.
